CloudWorldDemoClassification-AB.ipynb

```diff
@@ -302,7 +321,7 @@
     "\n",
     "connection_parameters = {\n",
     "    \"user_name\": \"admin\",\n",
-    "    \"password\": os.environ.get('ADB_PASSWORD'),\n",
+    "    \"password\": os.environ.get('ADB_ADMIN_PASSWORD'),\n",
     "    \"service_name\": os.environ.get('DATABASE_NAME') + \"_high\",\n",
```

Comment:
Please note that logging passwords would defeat the purpose of sending them as environment variables. So, it might not be a good idea to log passwords.

Reply:
I don't see how it would defeat the purpose. A user logged in to the session has full access to the environment variables anyway. Logging them will not introduce any new risk